lib/Config.ts

```diff
@@ -63,6 +63,7 @@ class Config {
     this.p2p = {
       listen: true,
       discover: true,
+      discoverInSeconds: 60 * 60, // 1 hour
```

Comment:
I think an hour is pretty short, I was thinking more like a day would make sense? Or maybe 12 hours? I think every hour can get pretty noisy and most of the time will not get us any new info.

Also, I think we can safely measure this in minutes instead of seconds, I doubt we need second precision. It's a little easier to calculate that way.

And for the name, I'm struggling to come up with something I really like but maybe `rediscoverTimer` or `rediscoverMinutes` might be a little clearer?

Reply:
Makes sense, but since discover's word meaning already something like; find something new, do we really want to rename these ?

Reply:
Well it's a new config option, I don't meant to rename any of the existing options. Here  I think that "rediscover" might make more sense because we're telling `xud` to continuously attempt to discover new nodes, e.g. repeating discovery, instead of only doing so on first connection.

Reply:
Most people won't know that `xud` discovers once in the beginning, so discover is fine with me. Rediscover raises more ? I'm afraid.

Reply:
We'll leave the existing option as `discover` for sure, but then the new option is tricky. I'd be ok with `discoverminutes` I suppose, it should definitely be all lower case tho to be consistent with the other config options.

Either way, this is an advanced option and not at all something people should have to mess with unless they have a good reason to, and definitely not if they don't know how the `discover` option works.